pages/learn/develop/runtime.md

```diff
@@ -70,6 +70,18 @@ DBUS_SYSTEM_BUS_ADDRESS=unix:path=/host_run/dbus/system_bus_socket
 
 Below you can find a couple of examples. All of them requires either prepending the command with the above `DBUS_SYSTEM_BUS_ADDRESS=...` or setting the variable for all commands by running `export DBUS_SYSTEM_BUS_ADDRESS=...` with the correct environment variable value from above.
 
+In multicontainer applications, you can also set `DBUS_SYSTEM_BUS_ADDRESS=unix:path=/host/run/dbus/system_bus_socket` in the `environment` section of your service in the `docker-compose.yml` file:
+
+```
```

Comment:
```suggestion
```yaml
```

While we don't make use of these I've been adding these as a matter of styling improvements.

Reply:
time for a styleguide laying this and other conventions out?

Reply:
Agreed. We should probably just adopt something such as [Google's](https://developers.google.com/style), and obviously we can supplement our own product names and other balena specific items.